image/etc/init.d/resize2fs_once

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
I think that this has to go someplace else.

The way it was coded before, it would get executed on the boot after prepare-base.sh was run. With this restructuring it will get executed on the boot before prepare-base.sh is run

Reply:
So, I thought about that, but I think this is ok. It's copied into the filesystem in the `prepare-base.sh` script, but it's not enabled. That's still done by the `add-gateway.sh` script, as before.

Reply:
Actually, I think it should be ok. The symlink is created in add-gateway.sh which is run after prepare-base.sh